modules/api/php/endpoints/candidate/visit/image/format.class.inc

```diff
@@ -90,7 +90,11 @@ class Format extends Endpoint
             $handler = new Format\Brainbrowser($this->_image);
             break;
         case 'thumbnail':
-            $handler = new Format\Thumbnail($this->_image);
+        case 'nifti':
```

Comment:
aren't thumbnails intended to be displayed inline?

(It looks like the code already had an attachment Content-Disposition, so I guess it doesn't matter..)

Reply:
yes, the thumbnail is mainly used to be displayed in the imaging browser but it was already available for download in the API and that is the file I used to base all the other classes so I kept it. @driusan does it answer your question?